### PR TITLE
[feature][doc] Add docs for the new S3 offloader

### PR DESCRIPTION
**Pull request by momo-jun**
### Modifications

1. Add docs for the new feature introduced in #15710.
2. Apply similar changes to the topic of the Aliyun offloader.

The preview looks good.
image
image
image
image
image
image
image
image

### Documentation

- [x] `doc`

### CODE REVIEW

**Comment by momo-jun** [comment]
Ping @zymap @hangc0276 @Anonymitaet for review.

File: site2/docs/tiered-storage-s3.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+:::
+
+
+### Configure S3 offloader driver
+
+You can configure the S3 offloader driver in the configuration file `broker.conf` or `standalone.conf`.
+
+- **Required** configurations are as below.
+  
+  | Required configuration | Description | Example value |
```
Comment (Anonymitaet) [inline site2/docs/tiered-storage-s3.md:60]: double check: example or default value?

Comment (momo-jun) [reply site2/docs/tiered-storage-s3.md:60]: Example value

**Review by Anonymitaet — APPROVED** [review]

File: site2/docs/tiered-storage-s3.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+This chapter guides you through every step of installing and configuring the S3 offloader and using it with Pulsar. 
+
+## Installation
+
+Follow the steps below to install the S3 offloader.
+
+### Prerequisite
+
+- Pulsar: 2.9.2 or later versions
```
Comment (zymap) [inline site2/docs/tiered-storage-s3.md:17]: 2.9.2 has been released. Let's fix it as 2.9.3

**Pull request merged**